src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -844,15 +844,10 @@ class Program
             Assert.False(comp.Assembly.RuntimeSupportsByRefFields);
 
             comp = CreateEmptyCompilation(source, references: new[] { refAB }, parseOptions: TestOptions.Regular10);
-            // https://github.com/dotnet/roslyn/issues/62131: Enable updated escape rules if
-            // System.Runtime.CompilerServices.RuntimeFeature.ByRefFields exists.
             comp.VerifyDiagnostics(
                 // (3,12): error CS8936: Feature 'ref fields' is not available in C# 10.0. Please use language version 11.0 or greater.
                 //     public ref T F;
-                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion10, "ref T").WithArguments("ref fields", "11.0").WithLocation(3, 12),
-                // (10,20): error CS8167: Cannot return by reference a member of parameter 'r' because it is not a ref or out parameter
-                //         return ref r.F;
-                Diagnostic(ErrorCode.ERR_RefReturnParameter2, "r").WithArguments("r").WithLocation(10, 20));
+                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion10, "ref T").WithArguments("ref fields", "11.0").WithLocation(3, 12));
```

Comment:
I think we're missing an error on line 854 below. If you try to use ref fields in a C# 11 compilation but the runtime feature flag doesn't exist, the compiler should complain.

This is described in the spec:
"This feature [ref fields] requires runtime support and changes to the ECMA spec. As such these will only be enabled when the corresponding feature flag is set in corelib. The issue tracking the exact API is tracked here dotnet/runtime#64165"

Reply:
Thanks. https://github.com/dotnet/roslyn/issues/62919.